File: benchmarks/tasks/filter_map/filter_map.js
// SPDX-License-Identifier: MIT-0

const result = Array.from({length: 10}, (_, i) => i + 1)
    .filter(x => x % 2 === 0)
    .map(x => x * x)
    .reduce((a, b) => a + b, 0);

console.log(result);

File: benchmarks/tasks/filter_map/filter_map.cpp
// SPDX-License-Identifier: MIT-0

#include <iostream>

int main() {
    int sum = 0;
    for (int x = 1; x <= 10; ++x) {
        if (x % 2 == 0) {
            sum += x * x;
        }
    }
    std::cout << sum << std::endl;
    return 0;
}
